default_middlewares.go

```diff
@@ -0,0 +1,44 @@
+package fuego
+
+import (
+	"log/slog"
+	"net/http"
+)
+
+type LoggingConfig struct {
+	Disabled        bool
```

Comment:
I would like to suggest this


```suggestion
	DisableLogging        bool
```

The Disabled would be nice if there were no other Disable without "d" aside.

I think it would be more readable